libs/sdk-core/src/persist/db.rs

```diff
@@ -190,6 +190,15 @@ impl SqliteStorage {
              
              DROP TABLE old_swaps;            
             "),
+            M::up(
+                "
+             CREATE TABLE IF NOT EXISTS payments_external_info (
+              payment_hash TEXT NOT NULL PRIMARY KEY,
```

Comment:
NIT:  In my last feedback I mentioned to change the payment_hash to payment_id so we can associate data later on non lightning payments (closed channel for example).

Reply:
Sorry I must have missed that. Done.